### PR TITLE
Added testhost process id and processname

### PR DESCRIPTION
**Pull request by harshjain2**
Added process id and process name when testhost debugging is enabled.

### CODE REVIEW

**Comment by harshjain2** [comment]
Snapshot : 
image

File: src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs [hunk 1]
```diff
@@ -121,15 +122,18 @@ public virtual void SetupChannel(IEnumerable<string> sources)
...
 
+                // Launch the test host.
+                var testHostProcessId = this.testHostManager.LaunchTestHost(testHostStartInfo);
+
                 // Warn the user that execution will wait for debugger attach.
                 var hostDebugEnabled = Environment.GetEnvironmentVariable("VSTEST_HOST_DEBUG");
                 if (!string.IsNullOrEmpty(hostDebugEnabled) && hostDebugEnabled.Equals("1", StringComparison.Ordinal))
                 {
                     ConsoleOutput.Instance.WriteLine(CrossPlatEngineResources.HostDebuggerWarning, OutputLevel.Warning);
+                    ConsoleOutput.Instance.WriteLine(
```
Comment (codito) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs]: nit: indent.

File: src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs [hunk 2]
```diff
@@ -121,15 +122,18 @@ public virtual void SetupChannel(IEnumerable<string> sources)
...
+                    string.Format("Process Id: {0}, Name: {1}", processId, Process.GetProcessById(testHostProcessId).ProcessName),
```
Comment (codito) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs]: Please don't use Process directly. Is there a core utilities abstraction for Process that can be used? Or something in PlatformAbstractions?

Comment (codito) [inline src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyOperationManager.cs]: Consider increasing timeout in Line 141 for debug scenarios.

**Review by codito — APPROVED** [review]
Approved with suggestions.

**Pull request merged**